packages/nextra/src/loader.ts

```diff
@@ -243,43 +257,38 @@ async function loader(
   const cssImport = OFFICIAL_THEMES.includes(theme)
     ? `import '${theme}/style.css'`
     : ''
-  const finalResult = transform
-    ? await transform(result, { route: pageNextRoute })
-    : result
   const stringifiedPageOpts = JSON.stringify(pageOpts)
+  const pageOptsChecksum = IS_PRODUCTION
+    ? "''"
+    : JSON.stringify(hashFnv32a(stringifiedPageOpts))
+  const dynamicMetaModules = dynamicMetaItems
+    .map(
+      descriptor =>
+        `[import(${JSON.stringify(descriptor.metaFilePath)}), ${JSON.stringify(
+          descriptor
+        )}]`
+    )
+    .join(',')
 
   return `import { setupNextraPage } from 'nextra/setup-page'
 import __nextra_layout from '${layout}'
 ${themeConfigImport}
 ${katexCssImport}
```

Comment:
these imports we can inject also in `_app.mdx` to yet reduce bundle size, but in another PR ;)